client/flutter/android/build.gradle

```diff
@@ -3,12 +3,19 @@ buildscript {
     repositories {
         google()
         jcenter()
+        // Repository for crashlytics
+        maven {
+            url 'https://maven.fabric.io/public'
+        }
     }
 
     dependencies {
         classpath 'com.android.tools.build:gradle:3.5.0'
         classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlin_version"
         classpath 'com.google.gms:google-services:4.3.3'
+        // Crashlytics dependencies
+        classpath 'com.google.gms:google-services:4.3.0'
```

Comment:
I think we should remove this line -- we already have google-services in the previous line

Reply:
Good catch